recipes/gfortran_impl_osx-64/meta.yaml

```diff
@@ -0,0 +1,142 @@
+{% set gfortran_version = [7, 3, 0] %}
+{% set libgfortran_version = [4, 0, 0] %}
+{% set max_libgfortran_version = "5.0.0.a0" %}
+{% set libquadmath_version = [0, 0, 0] %}
+{% set libgcc_s_version = [1, 0, 0] %}
+{% set chost = "x86_64-apple-darwin13.4.0" %}
+
+package:
+  name: gfortran_impl_osx-64
+  version: {{ gfortran_version|join('.') }}
+
+source:
+  url: https://ftp.gnu.org/gnu/gcc/gcc-{{ gfortran_version|join('.') }}/gcc-{{ gfortran_version|join('.') }}.tar.gz
+  sha256: fa06e455ca198ddc11ea4ddf2a394cf7cfb66aa7e0ab98cc1184189f1d405870
+  patches:
+    - libgcc_macosx_min_version.patch
+
+build:
+  number: 0
+  skip: True  # [not osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - libiconv {{ libiconv }}
+    - zlib {{ zlib }}
+    - gmp {{ gmp }}
+    - mpfr {{ mpfr }}
+    - isl {{ isl }}
+    - mpc ==1.1.0
```

Comment:
Can you remove the pinnings on the 6 packages above?

Reply:
These pick up the central conda forge pinnings. Is that not what we want?

Happy to remove them! Not at my machine right now though.

Reply:
You can replace `- isl {{ isl }}` with just `- isl`. It does the same thing

Reply:
Ahhh got it. Thanks!